carbon-design-system/carbon-addons-iot-react
# fix(table): prevent pagination from overflowling container

Closes #3278 

**Summary**

- Prevent table pagination from overflowing

**Change List (commits, features, bugs, etc)**

- Hide page count in pagination if its width less than `625px`

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3632--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table--with-pagination)
- Resize screen width till:
- Pagination container width `625px` - items per page hides
- Storybook container width `672px` - page select hides (Carbon pagination breakpoint listens to screen width)
- Pagination container width `500px` - all items hide except buttons

</br>

Previous
![Screenshot 2022-11-04 at 22 46 30](https://user-images.githubusercontent.com/112560065/200079776-e88f23ac-531c-4cd0-b4d1-86ef069f6f32.png)

Current
![Screenshot 2022-11-04 at 22 47 30](https://user-images.githubusercontent.com/112560065/200079921-521060e4-528a-4fc8-93ff-3ae60772e52c.png)




**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
